### PR TITLE
Add translations + prevent error when adding field

### PR DESCRIPTION
**Pull request by Rias500**
- Adds translations for "Opening Time" and "Closing Time"
- Prevents an error when adding the field to a Global Section

### CODE REVIEW

File: src/templates/input.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {% from "_includes/forms" import time %}
 <tr>
     <th>
-        {{ craft.i18n.getLocaleData().getWeekDayName(day) }}
+        {{ craft.i18n.getLocaleData().getWeekDayName(day) | ucfirst }}
```
Comment (brandonkelly) [inline src/templates/input.html]: Is there a place where this was necessary? Week day names should already be correctly capitalized.

File: src/fields/StoreHoursField.php [hunk 2]
```diff
@@ -57,6 +57,10 @@ public function getInputHtml($value, ElementInterface $element = null): string
      */
     public function serializeValue($value, ElementInterface $element = null)
     {
+        if (!$value) {
+            return null;
```
Comment (brandonkelly) [inline src/fields/StoreHoursField.php]: Let’s move this to a separate PR. Each PR should be focussed on a single feature/enhancement/bugfix. Thanks!

File: src/templates/input.html (hunk 1 above)
Comment (Rias500) [inline src/templates/input.html]: I guess officially in Ditch they're not capitalised but I thought it looked nicer, didn't mean to add this to the PR!

File: src/fields/StoreHoursField.php (hunk 2 above)
Comment (Rias500) [inline src/fields/StoreHoursField.php]: Will do! I'll update the PR tomorrow
